### ISSUE TITLE
[Servidor][Valgrind]

### ISSUE THREAD

**Issue by Marialuisaclaro** [IIC2133-PUC/2020-2#104]
Hola!
Tengo mac y puedo correr Valgrind ahí, pero no perfectamente. Quería saber si se puede ocupar el servidor para correr valgrind. En SO podíamos hacerlo, pero nose si acá será lo mismo.

Gracias!

**Comment by vichoeq** [comment]
En SO les daban acceso por ssh al servidor, no?

**Comment by Marialuisaclaro** [comment]
Si! Acá se puede hacer lo mismo?

**Comment by vichoeq** [comment]
No, ya que en el servidor tenemos almacenados los tests que se usan para corregir (no son los mismos que se ls entregan a ustedes)

Estamos trabajando en una forma de que puedan usar valgrind igual que como pueden testear en el servidor, pero tiene varios problemas que aun no resolvemos. Lo siento.
